src/schema/v2/Feature/FeatureMeta.ts

```diff
@@ -0,0 +1,35 @@
+import { GraphQLNonNull, GraphQLObjectType, GraphQLString } from "graphql"
+import { Gravity } from "types/runtime"
+import { ResolverContext } from "types/graphql"
+import { markdownToText } from "lib/helpers"
+
+const DEFAULT_META_DESCRIPTION =
+  "Artsy is the world’s largest online art marketplace. Browse over 1 million artworks by iconic and emerging artists, presented by 4000+ galleries and top auction houses in over 100 countries."
```

Comment:
It seems like this is the sort of thing that should probably be a consistent fallback in our meta tags and extracted to some central const accordingly.